modules/tensor_mechanics/doc/content/source/auxkernels/RankTwoScalarAux.md

```diff
@@ -267,6 +287,26 @@ is used as the argument for the `variable` input parameter in the `RankTwoScalar
 !listing modules/porous_flow/test/tests/thm_rehbinder/free_outer.i
          block=AuxVariables/stress_rr
 
+## Stress Intensity
+
+The scalar type `StressIntensity` calculates the stress intensity for a Rank-2 tensor, as shown in
+[eq:sint_scalar_type].
+\begin{equation}
+\label{eq:sint_scalar_type}
+s = 2 \sigma_{max}^{shear} = \sigma_{max}^{principal} - \sigma_{min}^{principal}
+\end{equation}
+
```

Comment:
Maybe worth adding here something like this:
`This quantity is useful in evaluating whether a Tresca failure criterion has been met, and is two times the MaxShear quantity.`

Reply:
Done.